Fitness/Views/Cells/ProCell.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  ProCell.swift
+//  Fitness
+//
+//  Created by Cameron Hamidi on 4/22/19.
+//  Copyright © 2019 Keivan Shahida. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class ProCell: UICollectionViewCell {
+    
+    // MARK: - INITIALIZATION
+    static let identifier = Identifiers.proCell
```

Comment:
Is this alias necessary?

Reply:
It's used for registering and dequeuing cells in the HomeController

Reply:
this shouldn't be necessary if you move the `collectionView` code into the file that contains the `collectionView`